packages/flutter/lib/src/painting/alignment.dart

```diff
@@ -621,7 +621,7 @@ class _MixedAlignment extends AlignmentGeometry {
 
   @override
   Alignment resolve(TextDirection direction) {
-    assert(direction != null);
+    assert(direction != null, 'Cannot resolve $runtimeType without a TextDirection.');
```

Comment:
Do we need test for this?

Reply:
The precise message? I can add a test if you like, sure. I hadn't added one for this because I wasn't particularly concerned about whether the message was changed.

Reply:
I think without a test is fine